Management/src/ActuatorApi/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "profiles": {
+    "http": {
```

Comment:
Why are there no https profiles?

Reply:
I ran into several issues regarding https and redirection, and the simplest solution was to hide the profile (because I couldn't get the containerized SBA server to trust the ASP.NET Core dev cert with hostname mismatch). Now I think the solution of actuators on a dedicated, non-https port and commenting out the `app.UseHttpsRedirection()` with a note why might be the simplest option.